CHANGELOG.md

```diff
@@ -406,6 +406,7 @@
 - Update importer date format to be MM/DD/YYYY
 - Separate intended beneficiaries with a pipe
 - Add a field to record the UK delivery partner named contact for project-level activities
+- Validate country codes against all valid-country codes
```

Comment:
Could we just be clear it is only the importer that has changed?

Reply:
Yeah, makes sense. Have updated this and the commit message to be clearer